lib/src/core/authentication-core.ts

```diff
@@ -439,7 +442,10 @@ export class AuthenticationCore<T> {
 
     public async getBasicUserInfo(): Promise<BasicUserInfo> {
         const sessionData = await this._dataLayer.getSessionData();
-        const authenticatedUser = AuthenticationUtils.getAuthenticatedUserInfo(sessionData?.id_token);
+        const authenticatedUser = AuthenticationUtils.getAuthenticatedUserInfo(
```

Comment:
I think it will be better to move this function from utils to helpers

Reply:
Fixed